ensembl_metadata/models/genome.py

```diff
@@ -0,0 +1,160 @@
+from django.db import models
+from ensembl_metadata.models.assembly import Assembly
+from ensembl_metadata.models.datarelease import DataRelease
+from ensembl_metadata.models.organism import Organism
+import uuid
+
+
+class Division(models.Model):
+    division_id = models.AutoField(primary_key=True)
+    name = models.CharField(unique=True, max_length=32)
+    short_name = models.CharField(unique=True, max_length=8)
+
+
+class Genome(models.Model):
+    genome_id = models.IntegerField(primary_key=True)
+    genome_uuid = models.CharField(max_length=128, default=uuid.uuid1, unique=True)
+    assembly = models.ForeignKey(Assembly, models.CASCADE, related_name='genomes')
+    organism = models.ForeignKey(Organism, models.CASCADE, related_name='genomes')
+    genebuild = models.CharField(max_length=255)
+    has_pan_compara = models.IntegerField()
+    has_variation = models.IntegerField()
+    has_microarray = models.IntegerField()
+    has_peptide_compara = models.IntegerField()
+    has_genome_alignments = models.IntegerField()
+    has_synteny = models.IntegerField()
+    has_other_alignments = models.IntegerField()
+    created = models.DateTimeField(auto_now_add=True)
+
+
+class GenomeDatabase(models.Model):
+    genome_database_id = models.AutoField(primary_key=True)
+    genome = models.ForeignKey(Genome, on_delete=models.CASCADE,
+                               related_name='databases')
+    dbname = models.CharField(max_length=64)
+    species_id = models.IntegerField()
+    type = models.CharField(max_length=13, blank=True, null=True)
+
+    class Meta:
+        unique_together = (('dbname', 'species_id', 'genome'),)
+
+
+class GenomeAlignment(models.Model):
+    genome_alignment_id = models.AutoField(primary_key=True)
+    genome = models.ForeignKey(Genome, on_delete=models.CASCADE,
+                               related_name='alignments')
+    type = models.CharField(max_length=32)
+    name = models.CharField(max_length=128)
+    count = models.IntegerField()
```

Comment:
Not that really really matter, but there is a PositiveIntegerField available as well. https://docs.djangoproject.com/en/3.1/ref/models/fields/#positiveintegerfield, and depending on expected values, we can even use https://docs.djangoproject.com/en/3.1/ref/models/fields/#positivebigintegerfield

Reply:
I'll leave this for now - I think it makes sense to merge this table with some of the other 'statistic' ones, and generalise from 'count' to a more generic type, which could be, e.g. a mean/average, requiring a decimal.

Reply:
Fair enough.